target/linux/ipq806x/base-files/lib/preinit/05_set_iface_mac_ipq806x.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
preinit is sourced, so it should be non-executable and the shebang should be removed.